CONTRIBUTING.md

```diff
@@ -311,6 +310,14 @@ The third command ensures to install the BIDS schema code as an "editable" insta
 so that if you make changes to the schema files,
 these are automatically reflected in the sourcecode.
 
+Note that if you need to work on the some of the Python code 
+that is used to render the specification, 
+you will probably have to also run:
+
+```bash
+pip install -e tools/schemacode[render]
+```
+
```

Comment:
```suggestion

This installs the `bidsschemacode` package in "editable" mode,
so that any changes you make to the code will be reflected when you use it,
such as when you build the documentation locally.

```